src/components/layout/Col/Col.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box } from 'reflexbox';
+import PropTypes from 'prop-types';
+
+import { pxToRem } from '../../../utils/helpers';
+import { ColProps, Cols } from './Col.types';
+
+const getColWidth = (cols: Cols): { flex: string } | { width: number } => {
+  if (cols === 'auto') return { flex: '1 1 auto' };
+
+  return { width: cols / 12 };
+};
+
+const Col: React.FC<ColProps> = ({ children, cols, offset }) => (
+  <Box ml={`${(100 / 12) * offset}%`} px={pxToRem(15)} {...getColWidth(cols)}>
```

Comment:
Imo it'd make sense for gutter (15) to be set in theme as well. Any app could then override it by their own theme